example/genqlient.yaml

```diff
@@ -1,11 +1,11 @@
 # These are the defaults, and are just included to be explicit.
 package: example
 schema: schema.graphql
-queries:
+operations:
```

Comment:
I notice that generation still works when this key is `queries` since `genqlient.graphql` is the default value for `operations`. What do you think about using `yaml.UnmarshalStrict` to detect invalid keys in the configuration?

Reply:
Yeah, that's probably a good idea.  This actually confused me for a while, because I copied this config for the integration tests, but actually wanted to change the value, and of course it didn't work.  I'll update it in a separate diff.